doc/tools/pkcs11-tool.1.xml

```diff
@@ -373,6 +373,19 @@
 					<listitem><para>Decrypt some data.</para></listitem>
 				</varlistentry>
 
+				<varlistentry>
+					<term>
+						<option>--encrypt</option>,
+					</term>
+					<listitem><para>Encrypt some data.</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>--Unwrap</option>,
```

Comment:
lowercase 'u'

Reply:
Thanks for the review, I'll fix it. 

But I have question.. Is command line format (for unwrap)  acceptable (especially use of `--application-id`  `--application-label`)?  Use key id 22 to unwrap file input.key as AES key with ID 90 and label `unwraped-key`:
```
pkcs11-tool --unwrap --mechanism RSA-PKCS --id 22 -i input.key --key-type AES: --application-id 90 --applicatin-label unwraped-key
```
If appropriate, I can add another switch for specification of the unwrapped key.

For the wrap operation, I have similar question..  I suggest the following command line format (use key id 22 to wrap key id 90):
```
pkcs11-tool --wrap --mechanism RSA-PKCS --id 22 --application-id 90 -o output.key
```

I'll add the  the wrap code soon...

Reply:
(sorry for a delay)
I do not like much the reuse of applicatin-id here. This is originally used for data objects. I think we will need a new switches for this not to confuse users. But I am not sure what would be the best terminology. `--new-id` and `--new-label` or `--wrapped-key-id` and `--wrapped-key-label` are just some ideas.

Reply:
I am aware that `--application_id`  and  `--application_label`is not used in its original meaning. As an alternative, I still like it the most:

a) unwrap: use the `--new-label` and `--new-id` switches (alternative `--new-key-label`, `--new-key-id`)
example:
```
pkcs11-tool --unwrap --mechanism RSA-PKCS --id 22 -i input.key --key-type AES: --new-key-id 90 --new-key-label unwraped-key

```
b) wrap:  it is possible to use the keyword `--wrap` with an argument, where either the `id` or `label` of the key that will be the subject of the wrapping can be used.

example:
```
pkcs11-tool --wrap 90 --mechanism RSA-PKCS --id 22  -o output.key

pkcs11-tool --wrap key_sign1  --mechanism RSA-PKCS --id 22  -o output.key
```

I would like other OpenSC developers to comment on this before applying this PR.

Thanks.

Reply:
@frankmorgner what do you think?